README.rst

```diff
@@ -38,8 +35,9 @@ learning projects:
 How DVC works
 =============
 
-We encourage you to read our `Get Started <https://dvc.org/doc/get-started>`_ guide to better understand what DVC
-is and how it can fit your scenarios.
+    We encourage you to read our `Get Started
+    <https://dvc.org/doc/get-started>`_ guides to better understand what DVC
+    does and how it can fit your scenarios.
```

Comment:
```suggestion
We encourage you to read our `Get Started <https://dvc.org/doc/get-started>`_ guide to better understand what DVC
is and how it can fit your scenarios.
```